File: dayFour/javaScript/After Break/TaskTen.js
const prompt = require("prompt-sync")();
let number = 0;
let sum = 0;
let average = 0;
let count = 1;
while(count <= 10){
    let number = Number(prompt("Enter a number: "));
    if(number < 0 || number > 100){
        console.log("Invalid input(Enter number between 1 to 100)");
    }
    else {
    sum += number;
    }
    count++;
}
average = sum / count ;
console.log("average is: " +  average);

File: dayFour/java/After Break/TaskTen.java
import java.util.Scanner;
public class TaskTen {
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        double number = 0;
        double sum = 0;
        int count = 1;
        while(count <= 10){
            System.out.print("Enter a number: ");
            number = input.nextDouble();
            if(number < 0 && number < 100){
                System.out.println("Invalid input(Enter number between 1 to 100)");
            }
            else {
            sum += number;
            count++;
            }
        }
        double average = sum / count;
        System.out.println("average is: " +  average);
        
    }
}
